src/main/resources/templates/db_scripts/inventory-hierarchy/addNullChecksToRecordsViewFunctions.sql

```diff
@@ -343,7 +343,13 @@ WITH
                                                                                                         'code', holdTempLoc.locJsonb ->> 'code',
                                                                                                         'campusName', holdTempLoc.locCampJsonb ->> 'name',
                                                                                                         'libraryName', holdTempLoc.locLibJsonb ->> 'name',
-                                                                                                        'institutionName', holdTempLoc.locInstJsonb ->> 'name'))
+                                                                                                        'institutionName', holdTempLoc.locInstJsonb ->> 'name'),
+                                                                                     'effectiveLocation',
+                                                                                     jsonb_build_object('name', COALESCE(holdEffLoc.locJsonb ->> 'discoveryDisplayName', holdEffLoc.locJsonb ->> 'name'),
+                                                                                                        'code', holdEffLoc.locJsonb ->> 'code',
+                                                                                                        'campusName', holdEffLoc.locCampJsonb ->> 'name',
+                                                                                                        'libraryName', holdEffLoc.locLibJsonb ->> 'name',
+                                                                                                        'institutionName', holdEffLoc.locInstJsonb ->> 'name'))
```

Comment:
Are these SQL files to be executed with a specific version in the `schema.json`?

As I understand it, these will have already have been run for a previous version and should not be changed, instead a new script introduced for the forthcoming version that re-defines the view.

(the same question applies to createRecordsViewFunction.sql however I've only commented on the first one)

Reply:
Ah, okay.  I will remove this and write a separate script.

Reply:
Thank you